src/core/NativeBaseProvider.tsx

```diff
@@ -94,7 +93,7 @@ const NativeBaseProvider = (props: NativeBaseProviderProps) => {
             <OverlayProvider isSSR>
               <ToastProvider>
                 <InitializeToastRef />
-                <SSRProvider>{children}</SSRProvider>
+                {children}
```

Comment:
might have to check react, react-native version, otherwise the old versions can not support

Reply:
very good